exercises/dominoes/dominoes.rb

```diff
@@ -0,0 +1,53 @@
+class Dominoes
+  require_relative 'domino'
+  attr_accessor :list
+
+  #[ArrayTuples], false => [Domino] or [Domino], true => [Domino]
+  def initialize( arr=[], already_dominoes= nil)
+    @list = []
+    arr.each do |item|
+      if !already_dominoes
+        @list << Domino.new(item[0], item[1])
+      else
+        @list << Domino.deep_copy(item)
```

Comment:
I fully admit to putting machinery in here that I am not 100% sure that I'll need! I've been sketching out a solution on paper during free moments on my commute, and that (unseen) material is sort of driving this code.